reana_job_controller/config.py

```diff
@@ -80,6 +80,9 @@
 should match `default_ccache_name` in krb5.conf.
 """
 
+KRB5_TOKEN_CACHE_FILENAME = 'krb5_1000'
```

Comment:
Usually, KRB5 ticket name has user ID in it ``krb5_%{uid}``. As we are planning to do a user ID discovery mechanism, so I would suggest doing something similar:

 ``KRB5_TOKEN_CACHE_FILENAME = 'krb5_{}'.format(WORKFLOW_RUNTIME_USER_UID)``